packages/react-components/src/components/SendBox.tsx

```diff
@@ -45,6 +45,10 @@ export interface SendBoxStrings {
    * The warning message when send box text length is more than max limit
    */
   textTooLong: string;
+  /**
+   * Aria label for send message button
+   */
+  sendButtonText: string;
```

Comment:
`sendButtonText` implies it would be a user facing text property (i.e. the text of the send button). Can this be `sendButtonAriaLabel`

Reply:
yup on it!